webapp/app.py

```diff
@@ -113,28 +145,31 @@ def logout():
     """
     Empty the session, used to logout.
```

Comment:
``` python3
"""
Logout by removing the `authorization_token` from the session
"""
```

Reply:
Did you simply ignore this?

Reply:
Sorry, I might have missed it with the revert, but it was done

Reply:
Ah fair enough